ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml

```diff
@@ -8,13 +8,28 @@ resources:
   '*':
     requests:
       cpu: 1000m
-      memory: 3Gi
+      memory: 1Gi
 tests:
 - artifact_dir: /tmp/artifacts
-  as: sanity
+  as: build
   commands: rpm-ostree --version
   container:
     from: bin
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: ./ci/unit.sh
+  container:
+    from: src
+- artifact_dir: /tmp/artifacts
+  as: codestyle
+  commands: ./ci/ci-commitmessage-submodules.sh && ./ci/codestyle.sh
```

Comment:
Hmm, we should fold `ci-commitmessage-submodules.sh` into the latter IMO so we can keep the entrypoints here simple to make it easier to tweak on the rpm-ostree side in the future.